clustered/integration-test/build.gradle

```diff
@@ -64,14 +64,9 @@ task copyServerLibs(type: Copy) {
 test {
   dependsOn copyServerLibs
   environment 'JAVA_HOME', testJava.javaHome
-  if (testJava.javaVersion.isJava9Compatible()) {
-    environment 'JAVA_OPTS', '--add-modules java.xml.bind'
-  }
   //If this directory does not exist, tests will fail with a cryptic assert failure
   systemProperty 'kitInstallationPath', "$unzipKit.destinationDir/${project(':clustered:clustered-dist').archivesBaseName}-$project.version-kit"
   // Uncomment to include client logging in console output
   // testLogging.showStandardStreams = true
-}.doFirst {
```

Comment:
I think there should also be changes to the clustered OSGi tests to enable them in Java 11 too.

Reply:
There is a problem there -
if (testJava.javaVersion.isJava9Compatible()) {
    //Deploying the jaxb-runtime as an OSGi module is currently impossible - don't ask... sigh.
    jvmArgs += ['--add-modules', 'java.xml.bind']
    //Servers need JAXB too.
    jvmArgs += ['-Dtc-server-opts=--add-modules java.xml.bind']
  }
and add modules can't work in java11.

Reply:
Have a look at the changes I made to the unclustered OSGi tests in 0c9e97f1df1b9e7265539556fb24094543e690fc.